src/views/list/single-featured.php

```diff
@@ -68,6 +61,14 @@
 </div><!-- .tribe-events-event-meta -->
 <?php do_action( 'tribe_events_after_the_meta' ) ?>
 
+	<!-- Event Cost -->
+<?php if ( tribe_get_cost() ) : ?>
+	<div class="tribe-events-event-cost featured-event">
+		<span class="ticket-cost"><?php echo tribe_get_cost( null, true ); ?></span>
+		<?php do_action( 'tribe_events_inside_cost' ) ?>
```

Comment:
Document this new hook or note that it is documented elsewhere (and cite the file), if you do this from within another template that also uses the hook.